[glib] Build error

Package: glib:x64-linux-dynamic -> 2.72.3

**Host Environment**

- Host: x64-linux
- Compiler: GNU 9.3.0
-    vcpkg-tool version: 2023-08-02-6d13efa755f9b5e101712d210199e4139b4c29f6
    vcpkg-scripts version: 1e9cf6939 2023-08-03 (4 天前)

**To Reproduce**

`vcpkg install `

**Failure logs**

```
-- Using cached GNOME-glib-2.72.3.tar.gz.
-- Cleaning sources at /home/vis/Dev/vcpkg/buildtrees/glib/src/2.72.3-eda9959d65.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/vis/Dev/vcpkg/downloads/GNOME-glib-2.72.3.tar.gz
-- Applying patch use-libiconv-on-windows.patch
-- Applying patch libintl.patch
-- Using source at /home/vis/Dev/vcpkg/buildtrees/glib/src/2.72.3-eda9959d65.clean
-- Getting CMake variables for x64-linux-dynamic-dbg
-- Getting CMake variables for x64-linux-dynamic-rel
-- Configuring x64-linux-dynamic-dbg
-- Configuring x64-linux-dynamic-dbg done
-- Configuring x64-linux-dynamic-rel
-- Configuring x64-linux-dynamic-rel done
-- Package x64-linux-dynamic-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /home/vis/Dev/vcpkg/downloads/tools/ninja/1.10.2-linux/ninja install -v
    Working Directory: /home/vis/Dev/vcpkg/buildtrees/glib/x64-linux-dynamic-dbg
    Error code: 1
    See logs for more information:
      /home/vis/Dev/vcpkg/buildtrees/glib/package-x64-linux-dynamic-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_install_meson.cmake:33 (vcpkg_execute_required_process)
  buildtrees/versioning_/versions/glib/abff8e76676673462462f949844d3369d93be61d/portfile.cmake:49 (vcpkg_install_meson)
  scripts/ports.cmake:147 (include)



```
<details><summary>/home/vis/Dev/vcpkg/buildtrees/glib/package-x64-linux-dynamic-dbg-out.log</summary>

```
[1/565] /usr/bin/cc -Iglib/libcharset/libcharset.a.p -Iglib/libcharset -I../src/2.72.3-eda9959d65.clean/glib/libcharset -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC '-DGLIB_CHARSETALIAS_DIR="/home/vis/Dev/vcpkg/packages/glib_x64-linux-dynamic/debug/lib"' -fvisibility=hidden -MD -MQ glib/libcharset/libcharset.a.p/localcharset.c.o -MF glib/libcharset/libcharset.a.p/localcharset.c.o.d -o glib/libcharset/libcharset.a.p/localcharset.c.o -c ../src/2.72.3-eda9959d65.clean/glib/libcharset/localcharset.c
[2/565] rm -f glib/libcharset/libcharset.a && /usr/bin/ar csrDT glib/libcharset/libcharset.a glib/libcharset/libcharset.a.p/localcharset.c.o
[3/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/deprecated_gcache.c.o -MF glib/libglib-2.0.so.0.7200.3.p/deprecated_gcache.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/deprecated_gcache.c.o -c ../src/2.72.3-eda9959d65.clean/glib/deprecated/gcache.c
[4/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/deprecated_gallocator.c.o -MF glib/libglib-2.0.so.0.7200.3.p/deprecated_gallocator.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/deprecated_gallocator.c.o -c ../src/2.72.3-eda9959d65.clean/glib/deprecated/gallocator.c
...
Skipped 76 lines
...

[81/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gunidecomp.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gunidecomp.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gunidecomp.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gunidecomp.c
[82/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gutils.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gutils.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gutils.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gutils.c
[83/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gversion.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gversion.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gversion.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gversion.c
[84/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gvarianttypeinfo.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gvarianttypeinfo.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gvarianttypeinfo.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gvarianttypeinfo.c
[85/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gvariant.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gvariant.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gvariant.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gvariant.c
[86/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gvariant-parser.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gvariant-parser.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gvariant-parser.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gvariant-parser.c
[87/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gprintf.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gprintf.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gprintf.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gprintf.c
[88/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gvarianttype.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gvarianttype.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gvarianttype.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gvarianttype.c
[89/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gwakeup.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gwakeup.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gwakeup.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gwakeup.c
[90/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gvariant-serialiser.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gvariant-serialiser.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gvariant-serialiser.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gvariant-serialiser.c
[91/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gthread-posix.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gthread-posix.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gthread-posix.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gthread-posix.c
[92/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/glib-unix.c.o -MF glib/libglib-2.0.so.0.7200.3.p/glib-unix.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/glib-unix.c.o -c ../src/2.72.3-eda9959d65.clean/glib/glib-unix.c
[93/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/giounix.c.o -MF glib/libglib-2.0.so.0.7200.3.p/giounix.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/giounix.c.o -c ../src/2.72.3-eda9959d65.clean/glib/giounix.c
[94/565] /usr/bin/cc -Iglib/libglib-2.0.so.0.7200.3.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../../include -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -pthread '-DG_LOG_DOMAIN="GLib"' -DGLIB_COMPILATION -fvisibility=hidden -MD -MQ glib/libglib-2.0.so.0.7200.3.p/gspawn.c.o -MF glib/libglib-2.0.so.0.7200.3.p/gspawn.c.o.d -o glib/libglib-2.0.so.0.7200.3.p/gspawn.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gspawn.c
[95/565] /usr/bin/cc -Igio/xdgmime/libxdgmime.a.p -Igio/xdgmime -I../src/2.72.3-eda9959d65.clean/gio/xdgmime -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -DHAVE_CONFIG_H -DXDG_PREFIX=_gio_xdg -fvisibility=hidden -MD -MQ gio/xdgmime/libxdgmime.a.p/xdgmimealias.c.o -MF gio/xdgmime/libxdgmime.a.p/xdgmimealias.c.o.d -o gio/xdgmime/libxdgmime.a.p/xdgmimealias.c.o -c ../src/2.72.3-eda9959d65.clean/gio/xdgmime/xdgmimealias.c
[96/565] /usr/bin/cc -Iglib/gtester.p -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -UG_DISABLE_ASSERT -MD -MQ glib/gtester.p/gtester.c.o -MF glib/gtester.p/gtester.c.o.d -o glib/gtester.p/gtester.c.o -c ../src/2.72.3-eda9959d65.clean/glib/gtester.c
[97/565] /usr/bin/cc -Igthread/libgthread-2.0.so.0.7200.3.p -Igthread -I../src/2.72.3-eda9959d65.clean/gthread -I. -I../src/2.72.3-eda9959d65.clean -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC '-DG_LOG_DOMAIN="GThread"' -fvisibility=hidden -MD -MQ gthread/libgthread-2.0.so.0.7200.3.p/gthread-impl.c.o -MF gthread/libgthread-2.0.so.0.7200.3.p/gthread-impl.c.o.d -o gthread/libgthread-2.0.so.0.7200.3.p/gthread-impl.c.o -c ../src/2.72.3-eda9959d65.clean/gthread/gthread-impl.c
[98/565] /usr/bin/cc  -o glib/libglib-2.0.so.0.7200.3 glib/libglib-2.0.so.0.7200.3.p/deprecated_gallocator.c.o glib/libglib-2.0.so.0.7200.3.p/deprecated_gcache.c.o glib/libglib-2.0.so.0.7200.3.p/deprecated_gcompletion.c.o glib/libglib-2.0.so.0.7200.3.p/deprecated_grel.c.o glib/libglib-2.0.so.0.7200.3.p/deprecated_gthread-deprecated.c.o glib/libglib-2.0.so.0.7200.3.p/garcbox.c.o glib/libglib-2.0.so.0.7200.3.p/garray.c.o glib/libglib-2.0.so.0.7200.3.p/gasyncqueue.c.o glib/libglib-2.0.so.0.7200.3.p/gatomic.c.o glib/libglib-2.0.so.0.7200.3.p/gbacktrace.c.o glib/libglib-2.0.so.0.7200.3.p/gbase64.c.o glib/libglib-2.0.so.0.7200.3.p/gbitlock.c.o glib/libglib-2.0.so.0.7200.3.p/gbookmarkfile.c.o glib/libglib-2.0.so.0.7200.3.p/gbytes.c.o glib/libglib-2.0.so.0.7200.3.p/gcharset.c.o glib/libglib-2.0.so.0.7200.3.p/gchecksum.c.o glib/libglib-2.0.so.0.7200.3.p/gconvert.c.o glib/libglib-2.0.so.0.7200.3.p/gdataset.c.o glib/libglib-2.0.so.0.7200.3.p/gdate.c.o glib/libglib-2.0.so.0.7200.3.p/gdatetime.c.o glib/libglib-2.0.so.0.7200.3.p/gdir.c.o glib/libglib-2.0.so.0.7200.3.p/genviron.c.o glib/libglib-2.0.so.0.7200.3.p/gerror.c.o glib/libglib-2.0.so.0.7200.3.p/gfileutils.c.o glib/libglib-2.0.so.0.7200.3.p/ggettext.c.o glib/libglib-2.0.so.0.7200.3.p/ghash.c.o glib/libglib-2.0.so.0.7200.3.p/ghmac.c.o glib/libglib-2.0.so.0.7200.3.p/ghook.c.o glib/libglib-2.0.so.0.7200.3.p/ghostutils.c.o glib/libglib-2.0.so.0.7200.3.p/giochannel.c.o glib/libglib-2.0.so.0.7200.3.p/gkeyfile.c.o glib/libglib-2.0.so.0.7200.3.p/glib-init.c.o glib/libglib-2.0.so.0.7200.3.p/glib-private.c.o glib/libglib-2.0.so.0.7200.3.p/glist.c.o glib/libglib-2.0.so.0.7200.3.p/gmain.c.o glib/libglib-2.0.so.0.7200.3.p/gmappedfile.c.o glib/libglib-2.0.so.0.7200.3.p/gmarkup.c.o glib/libglib-2.0.so.0.7200.3.p/gmem.c.o glib/libglib-2.0.so.0.7200.3.p/gmessages.c.o glib/libglib-2.0.so.0.7200.3.p/gnode.c.o glib/libglib-2.0.so.0.7200.3.p/goption.c.o glib/libglib-2.0.so.0.7200.3.p/gpattern.c.o glib/libglib-2.0.so.0.7200.3.p/gpoll.c.o glib/libglib-2.0.so.0.7200.3.p/gprimes.c.o glib/libglib-2.0.so.0.7200.3.p/gqsort.c.o glib/libglib-2.0.so.0.7200.3.p/gquark.c.o glib/libglib-2.0.so.0.7200.3.p/gqueue.c.o glib/libglib-2.0.so.0.7200.3.p/grand.c.o glib/libglib-2.0.so.0.7200.3.p/grcbox.c.o glib/libglib-2.0.so.0.7200.3.p/grefcount.c.o glib/libglib-2.0.so.0.7200.3.p/grefstring.c.o glib/libglib-2.0.so.0.7200.3.p/gregex.c.o glib/libglib-2.0.so.0.7200.3.p/gscanner.c.o glib/libglib-2.0.so.0.7200.3.p/gsequence.c.o glib/libglib-2.0.so.0.7200.3.p/gshell.c.o glib/libglib-2.0.so.0.7200.3.p/gslice.c.o glib/libglib-2.0.so.0.7200.3.p/gslist.c.o glib/libglib-2.0.so.0.7200.3.p/gstdio.c.o glib/libglib-2.0.so.0.7200.3.p/gstrfuncs.c.o glib/libglib-2.0.so.0.7200.3.p/gstring.c.o glib/libglib-2.0.so.0.7200.3.p/gstringchunk.c.o glib/libglib-2.0.so.0.7200.3.p/gstrvbuilder.c.o glib/libglib-2.0.so.0.7200.3.p/gtestutils.c.o glib/libglib-2.0.so.0.7200.3.p/gthread.c.o glib/libglib-2.0.so.0.7200.3.p/gthreadpool.c.o glib/libglib-2.0.so.0.7200.3.p/gtimer.c.o glib/libglib-2.0.so.0.7200.3.p/gtimezone.c.o glib/libglib-2.0.so.0.7200.3.p/gtrace.c.o glib/libglib-2.0.so.0.7200.3.p/gtranslit.c.o glib/libglib-2.0.so.0.7200.3.p/gtrashstack.c.o glib/libglib-2.0.so.0.7200.3.p/gtree.c.o glib/libglib-2.0.so.0.7200.3.p/guniprop.c.o glib/libglib-2.0.so.0.7200.3.p/gutf8.c.o glib/libglib-2.0.so.0.7200.3.p/gunibreak.c.o glib/libglib-2.0.so.0.7200.3.p/gunicollate.c.o glib/libglib-2.0.so.0.7200.3.p/gunidecomp.c.o glib/libglib-2.0.so.0.7200.3.p/guri.c.o glib/libglib-2.0.so.0.7200.3.p/gutils.c.o glib/libglib-2.0.so.0.7200.3.p/guuid.c.o glib/libglib-2.0.so.0.7200.3.p/gvariant.c.o glib/libglib-2.0.so.0.7200.3.p/gvariant-core.c.o glib/libglib-2.0.so.0.7200.3.p/gvariant-parser.c.o glib/libglib-2.0.so.0.7200.3.p/gvariant-serialiser.c.o glib/libglib-2.0.so.0.7200.3.p/gvarianttypeinfo.c.o glib/libglib-2.0.so.0.7200.3.p/gvarianttype.c.o glib/libglib-2.0.so.0.7200.3.p/gversion.c.o glib/libglib-2.0.so.0.7200.3.p/gwakeup.c.o glib/libglib-2.0.so.0.7200.3.p/gprintf.c.o glib/libglib-2.0.so.0.7200.3.p/glib-unix.c.o glib/libglib-2.0.so.0.7200.3.p/gspawn.c.o glib/libglib-2.0.so.0.7200.3.p/giounix.c.o glib/libglib-2.0.so.0.7200.3.p/gthread-posix.c.o -L/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib -Wl,--as-needed -Wl,--no-undefined -shared -fPIC -Wl,--start-group -Wl,-soname,libglib-2.0.so.0 -Wl,-rpath,/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../lib -Wl,-rpath-link,/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../lib glib/libcharset/libcharset.a -Wl,-z,nodelete -Wl,-Bsymbolic-functions /home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/debug/lib/pkgconfig/../../lib/libpcre.so -pthread -lm -Wl,--end-group
[99/565] /usr/bin/python3 gio/gdbus-2.0/codegen/gdbus-codegen --interface-prefix org. --output-directory gio --generate-c-code gdbus-daemon-generated --c-namespace _G ../src/2.72.3-eda9959d65.clean/gio/dbus-daemon.xml
[100/565] /usr/bin/python3 gio/gdbus-2.0/codegen/gdbus-codegen --interface-prefix org.freedesktop.portal. --output-directory gio --generate-c-code xdp-dbus --c-namespace GXdp ../src/2.72.3-eda9959d65.clean/gio/org.freedesktop.portal.Documents.xml ../src/2.72.3-eda9959d65.clean/gio/org.freedesktop.portal.OpenURI.xml ../src/2.72.3-eda9959d65.clean/gio/org.freedesktop.portal.ProxyResolver.xml ../src/2.72.3-eda9959d65.clean/gio/org.freedesktop.portal.Trash.xml
[101/565] /usr/bin/cc -Igmodule/libgmodule-2.0.so.0.7200.3.p -Igmodule -I../src/2.72.3-eda9959d65.clean/gmodule -I. -I../src/2.72.3-eda9959d65.clean -Iglib -I../src/2.72.3-eda9959d65.clean/glib -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC '-DG_LOG_DOMAIN="GModule"' -fvisibility=hidden -MD -MQ gmodule/libgmodule-2.0.so.0.7200.3.p/gmodule.c.o -MF gmodule/libgmodule-2.0.so.0.7200.3.p/gmodule.c.o.d -o gmodule/libgmodule-2.0.so.0.7200.3.p/gmodule.c.o -c ../src/2.72.3-eda9959d65.clean/gmodule/gmodule.c
[102/565] /usr/bin/cc -Igio/xdgmime/libxdgmime.a.p -Igio/xdgmime -I../src/2.72.3-eda9959d65.clean/gio/xdgmime -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -DHAVE_CONFIG_H -DXDG_PREFIX=_gio_xdg -fvisibility=hidden -MD -MQ gio/xdgmime/libxdgmime.a.p/xdgmime.c.o -MF gio/xdgmime/libxdgmime.a.p/xdgmime.c.o.d -o gio/xdgmime/libxdgmime.a.p/xdgmime.c.o -c ../src/2.72.3-eda9959d65.clean/gio/xdgmime/xdgmime.c
[103/565] /usr/bin/cc -Igio/xdgmime/libxdgmime.a.p -Igio/xdgmime -I../src/2.72.3-eda9959d65.clean/gio/xdgmime -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -DHAVE_CONFIG_H -DXDG_PREFIX=_gio_xdg -fvisibility=hidden -MD -MQ gio/xdgmime/libxdgmime.a.p/xdgmimeicon.c.o -MF gio/xdgmime/libxdgmime.a.p/xdgmimeicon.c.o.d -o gio/xdgmime/libxdgmime.a.p/xdgmimeicon.c.o -c ../src/2.72.3-eda9959d65.clean/gio/xdgmime/xdgmimeicon.c
[104/565] /usr/bin/cc -Igio/xdgmime/libxdgmime.a.p -Igio/xdgmime -I../src/2.72.3-eda9959d65.clean/gio/xdgmime -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -DHAVE_CONFIG_H -DXDG_PREFIX=_gio_xdg -fvisibility=hidden -MD -MQ gio/xdgmime/libxdgmime.a.p/xdgmimeint.c.o -MF gio/xdgmime/libxdgmime.a.p/xdgmimeint.c.o.d -o gio/xdgmime/libxdgmime.a.p/xdgmimeint.c.o -c ../src/2.72.3-eda9959d65.clean/gio/xdgmime/xdgmimeint.c
[105/565] /usr/bin/cc -Igio/xdgmime/libxdgmime.a.p -Igio/xdgmime -I../src/2.72.3-eda9959d65.clean/gio/xdgmime -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -DHAVE_CONFIG_H -DXDG_PREFIX=_gio_xdg -fvisibility=hidden -MD -MQ gio/xdgmime/libxdgmime.a.p/xdgmimeglob.c.o -MF gio/xdgmime/libxdgmime.a.p/xdgmimeglob.c.o.d -o gio/xdgmime/libxdgmime.a.p/xdgmimeglob.c.o -c ../src/2.72.3-eda9959d65.clean/gio/xdgmime/xdgmimeglob.c
[106/565] /usr/bin/cc -Igio/xdgmime/libxdgmime.a.p -Igio/xdgmime -I../src/2.72.3-eda9959d65.clean/gio/xdgmime -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -DHAVE_CONFIG_H -DXDG_PREFIX=_gio_xdg -fvisibility=hidden -MD -MQ gio/xdgmime/libxdgmime.a.p/xdgmimecache.c.o -MF gio/xdgmime/libxdgmime.a.p/xdgmimecache.c.o.d -o gio/xdgmime/libxdgmime.a.p/xdgmimecache.c.o -c ../src/2.72.3-eda9959d65.clean/gio/xdgmime/xdgmimecache.c
[107/565] /home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/tools/gettext/bin/msgfmt ../src/2.72.3-eda9959d65.clean/po/it.po -o po/it/LC_MESSAGES/glib20.mo
FAILED: po/it/LC_MESSAGES/glib20.mo 
/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/tools/gettext/bin/msgfmt ../src/2.72.3-eda9959d65.clean/po/it.po -o po/it/LC_MESSAGES/glib20.mo
/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/tools/gettext/bin/msgfmt: error while loading shared libraries: libgettextsrc-0.21.so: cannot open shared object file: No such file or directory
[108/565] /usr/bin/cc -Igio/xdgmime/libxdgmime.a.p -Igio/xdgmime -I../src/2.72.3-eda9959d65.clean/gio/xdgmime -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -DHAVE_CONFIG_H -DXDG_PREFIX=_gio_xdg -fvisibility=hidden -MD -MQ gio/xdgmime/libxdgmime.a.p/xdgmimeparent.c.o -MF gio/xdgmime/libxdgmime.a.p/xdgmimeparent.c.o.d -o gio/xdgmime/libxdgmime.a.p/xdgmimeparent.c.o -c ../src/2.72.3-eda9959d65.clean/gio/xdgmime/xdgmimeparent.c
[109/565] /usr/bin/cc -Igio/xdgmime/libxdgmime.a.p -Igio/xdgmime -I../src/2.72.3-eda9959d65.clean/gio/xdgmime -I. -I../src/2.72.3-eda9959d65.clean -I/home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu99 -O0 -g -D_GNU_SOURCE -fno-strict-aliasing -DG_ENABLE_DEBUG -Wduplicated-branches -Wimplicit-fallthrough -Wmisleading-indentation -Wunused -Wno-unused-parameter -Wno-cast-function-type -Wno-pedantic -Wno-format-zero-length -Wno-variadic-macros -Werror=format=2 -Werror=init-self -Werror=missing-include-dirs -Werror=pointer-arith -Wstrict-prototypes -Wno-bad-function-cast -Werror=declaration-after-statement -Werror=implicit-function-declaration -Werror=missing-prototypes -fPIC -g -fPIC -DHAVE_CONFIG_H -DXDG_PREFIX=_gio_xdg -fvisibility=hidden -MD -MQ gio/xdgmime/libxdgmime.a.p/xdgmimemagic.c.o -MF gio/xdgmime/libxdgmime.a.p/xdgmimemagic.c.o.d -o gio/xdgmime/libxdgmime.a.p/xdgmimemagic.c.o -c ../src/2.72.3-eda9959d65.clean/gio/xdgmime/xdgmimemagic.c
[110/565] /usr/bin/python3 /home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/tools/meson/meson.py --internal symbolextractor /home/vis/Dev/vcpkg/buildtrees/glib/x64-linux-dynamic-dbg glib/libglib-2.0.so.0.7200.3 glib/libglib-2.0.so.0.7200.3 glib/libglib-2.0.so.0.7200.3.p/libglib-2.0.so.0.7200.3.symbols 
[111/565] /usr/bin/python3 /home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/tools/meson/meson.py --internal exe --capture gobject/glib-enumtypes.h -- /usr/bin/python3 gobject/glib-mkenums --template ../src/2.72.3-eda9959d65.clean/gobject/glib-enumtypes.h.template ../src/2.72.3-eda9959d65.clean/gobject/../glib/gunicode.h
[112/565] /usr/bin/python3 /home/vis/Dev/3dreconstruction/build/vcpkg_installed/x64-linux-dynamic/tools/meson/meson.py --internal exe --capture gio/gioenumtypes.h -- /usr/bin/python3 gobject/glib-mkenums --template ../src/2.72.3-eda9959d65.clean/gio/gioenumtypes.h.template ../src/2.72.3-eda9959d65.clean/gio/gappinfo.h ../src/2.72.3-eda9959d65.clean/gio/gasyncinitable.h ../src/2.72.3-eda9959d65.clean/gio/gasyncresult.h ../src/2.72.3-eda9959d65.clean/gio/gbufferedinputstream.h ../src/2.72.3-eda9959d65.clean/gio/gbufferedoutputstream.h ../src/2.72.3-eda9959d65.clean/gio/gbytesicon.h ../src/2.72.3-eda9959d65.clean/gio/gcancellable.h ../src/2.72.3-eda9959d65.clean/gio/gcontenttype.h ../src/2.72.3-eda9959d65.clean/gio/gcharsetconverter.h ../src/2.72.3-eda9959d65.clean/gio/gconverter.h ../src/2.72.3-eda9959d65.clean/gio/gconverterinputstream.h ../src/2.72.3-eda9959d65.clean/gio/gconverteroutputstream.h ../src/2.72.3-eda9959d65.clean/gio/gdatagrambased.h ../src/2.72.3-eda9959d65.clean/gio/gdatainputstream.h ../src/2.72.3-eda9959d65.clean/gio/gdataoutputstream.h ../src/2.72.3-eda9959d65.clean/gio/gdebugcontroller.h ../src/2.72.3-eda9959d65.clean/gio/gdebugcontrollerdbus.h ../src/2.72.3-eda9959d65.clean/gio/gdrive.h ../src/2.72.3-eda9959d65.clean/gio/gemblem.h ../src/2.72.3-eda9959d65.clean/gio/gemblemedicon.h ../src/2.72.3-eda9959d65.clean/gio/gfile.h ../src/2.72.3-eda9959d65.clean/gio/gfileattribute.h ../src/2.72.3-eda9959d65.clean/gio/gfileenumerator.h ../src/2.72.3-eda9959d65.clean/gio/gfileicon.h ../src/2.72.3-eda9959d65.clean/gio/gfileinfo.h ../src/2.72.3-eda9959d65.clean/gio/gfileinputstream.h ../src/2.72.3-eda9959d65.clean/gio/gfilemonitor.h ../src/2.72.3-eda9959d65.clean/gio/gfilenamecompleter.h ../src/2.72.3-eda9959d65.clean/gio/gfileoutputstream.h ../src/2.72.3-eda9959d65.clean/gio/gfileiostream.h ../src/2.72.3-eda9959d65.clean/gio/gfilterinputstream.h ../src/2.72.3-eda9959d65.clean/gio/gfilteroutputstream.h ../src/2.72.3-eda9959d65.clean/gio/gicon.h ../src/2.72.3-eda9959d65.clean/gio/ginetaddress.h ../src/2.72.3-eda9959d65.clean/gio/ginetaddressmask.h ../src/2.72.3-eda9959d65.clean/gio/ginetsocketaddress.h ../src/2.72.3-eda9959d65.clean/gio/ginitable.h ../src/2.72.3-eda9959d65.clean/gio/ginputstream.h ../src/2.72.3-eda9959d65.clean/gio/gio.h ../src/2.72.3-eda9959d65.clean/gio/gio-autocleanups.h ../src/2.72.3-eda9959d65.clean/gio/gioenums.h ../src/2.72.3-eda9959d65.clean/gio/gioerror.h ../src/2.72.3-eda9959d65.clean/gio/giomodule.h ../src/2.72.3-eda9959d65.clean/gio/gioscheduler.h ../src/2.72.3-eda9959d65.clean/gio/giostream.h ../src/2.72.3-eda9959d65.clean/gio/giotypes.h ../src/2.72.3-eda9959d65.clean/gio/gloadableicon.h ../src/2.72.3-eda9959d65.clean/gio/gmount.h ../src/2.72.3-eda9959d65.clean/gio/gmemoryinputstream.h ../src/2.72.3-eda9959d65.clean/gio/gmemorymonitor.h ../src/2.72.3-eda9959d65.clean/gio/gmemoryoutputstream.h ../src/2.72.3-eda9959d65.clean/gio/gmountoperation.h ../src/2.72.3-eda9959d65.clean/gio/gnativesocketaddress.h ../src/2.72.3-eda9959d65.clean/gio/gnativevolumemonitor.h ../src/2.72.3-eda9959d65.clean/gio/gnetworkaddress.h ../src/2.72.3-eda9959d65.clean/gio/gnetworkmonitor.h ../src/2.72.3-eda9959d65.clean/gio/gnetworkservice.h ../src/2.72.3-eda9959d65.clean/gio/goutputstream.h ../src/2.72.3-eda9959d65.clean/gio/gpermission.h ../src/2.72.3-eda9959d65.clean/gio/gpollableinputstream.h ../src/2.72.3-eda9959d65.clean/gio/gpollableoutputstream.h ../src/2.72.3-eda9959d65.clean/gio/gpollableutils.h ../src/2.72.3-eda9959d65.clean/gio/gpowerprofilemonitor.h ../src/2.72.3-eda9959d65.clean/gio/gproxy.h ../src/2.72.3-eda9959d65.clean/gio/gproxyaddress.h ../src/2.72.3-eda9959d65.clean/gio/gproxyaddressenumerator.h ../src/2.72.3-eda9959d65.clean/gio/gproxyresolver.h ../src/2.72.3-eda9959d65.clean/gio/gresolver.h ../src/2.72.3-eda9959d65.clean/gio/gresource.h ../src/2.72.3-eda9959d65.clean/gio/gseekable.h ../src/2.72.3-eda9959d65.clean/gio/gsimpleasyncresult.h ../src/2.72.3-eda9959d65.clean/gio/gsimpleiostream.h ../src/2.72.3-eda9959d65.clean/gio/gsimplepermission.h ../src/2.72.3-eda9959d65.clean/gio/gsimpleproxyresolver.h ../src/2.72.3-eda9959d65.clean/gio/gsocket.h ../src/2.72.3-eda9959d65.clean/gio/gsocketaddress.h ../src/2.72.3-eda9959d65.clean/gio/gsocketaddressenumerator.h ../src/2.72.3-eda9959d65.clean/gio/gsocketclient.h ../src/2.72.3-eda9959d65.clean/gio/gsocketconnectable.h ../src/2.72.3-eda9959d65.clean/gio/gsocketconnection.h ../src/2.72.3-eda9959d65.clean/gio/gsocketcontrolmessage.h ../src/2.72.3-eda9959d65.clean/gio/gsocketlistener.h ../src/2.72.3-eda9959d65.clean/gio/gsocketservice.h ../src/2.72.3-eda9959d65.clean/gio/gsrvtarget.h ../src/2.72.3-eda9959d65.clean/gio/gsubprocess.h ../src/2.72.3-eda9959d65.clean/gio/gsubprocesslauncher.h ../src/2.72.3-eda9959d65.clean/gio/gtask.h ../src/2.72.3-eda9959d65.clean/gio/gtcpconnection.h ../src/2.72.3-eda9959d65.clean/gio/gtcpwrapperconnection.h ../src/2.72.3-eda9959d65.clean/gio/gthemedicon.h ../src/2.72.3-eda9959d65.clean/gio/gthreadedsocketservice.h ../src/2.72.3-eda9959d65.clean/gio/gtlsbackend.h ../src/2.72.3-eda9959d65.clean/gio/gtlscertificate.h ../src/2.72.3-eda9959d65.clean/gio/gtlsclientconnection.h ../src/2.72.3-eda9959d65.clean/gio/gtlsconnection.h ../src/2.72.3-eda9959d65.clean/gio/gtlsdatabase.h ../src/2.72.3-eda9959d65.clean/gio/gtlsfiledatabase.h ../src/2.72.3-eda9959d65.clean/gio/gtlsinteraction.h ../src/2.72.3-eda9959d65.clean/gio/gtlspassword.h ../src/2.72.3-eda9959d65.clean/gio/gtlsserverconnection.h ../src/2.72.3-eda9959d65.clean/gio/gdtlsconnection.h ../src/2.72.3-eda9959d65.clean/gio/gdtlsclientconnection.h ../src/2.72.3-eda9959d65.clean/gio/gdtlsserverconnection.h ../src/2.72.3-eda9959d65.clean/gio/gunixconnection.h ../src/2.72.3-eda9959d65.clean/gio/gunixcredentialsmessage.h ../src/2.72.3-eda9959d65.clean/gio/gunixsocketaddress.h ../src/2.72.3-eda9959d65.clean/gio/gvfs.h ../src/2.72.3-eda9959d65.clean/gio/gvolume.h ../src/2.72.3-eda9959d65.clean/gio/gvolumemonitor.h ../src/2.72.3-eda9959d65.clean/gio/gzlibcompressor.h ../src/2.72.3-eda9959d65.clean/gio/gzlibdecompressor.h ../src/2.72.3-eda9959d65.clean/gio/glistmodel.h ../src/2.72.3-eda9959d65.clean/gio/gliststore.h ../src/2.72.3-eda9959d65.clean/gio/gapplication.h ../src/2.72.3-eda9959d65.clean/gio/gapplicationcommandline.h ../src/2.72.3-eda9959d65.clean/gio/gactiongroup.h ../src/2.72.3-eda9959d65.clean/gio/gactionmap.h ../src/2.72.3-eda9959d65.clean/gio/gsimpleactiongroup.h ../src/2.72.3-eda9959d65.clean/gio/gremoteactiongroup.h ../src/2.72.3-eda9959d65.clean/gio/gactiongroupexporter.h ../src/2.72.3-eda9959d65.clean/gio/gdbusactiongroup.h ../src/2.72.3-eda9959d65.clean/gio/gaction.h ../src/2.72.3-eda9959d65.clean/gio/gpropertyaction.h ../src/2.72.3-eda9959d65.clean/gio/gsimpleaction.h ../src/2.72.3-eda9959d65.clean/gio/gmenumodel.h ../src/2.72.3-eda9959d65.clean/gio/gmenu.h ../src/2.72.3-eda9959d65.clean/gio/gmenuexporter.h ../src/2.72.3-eda9959d65.clean/gio/gdbusmenumodel.h ../src/2.72.3-eda9959d65.clean/gio/gnotification.h ../src/2.72.3-eda9959d65.clean/gio/gsettingsbackend.h ../src/2.72.3-eda9959d65.clean/gio/gsettingsschema.h ../src/2.72.3-eda9959d65.clean/gio/gsettings.h ../src/2.72.3-eda9959d65.clean/gio/gdbusauthobserver.h ../src/2.72.3-eda9959d65.clean/gio/gcredentials.h ../src/2.72.3-eda9959d65.clean/gio/gdbusutils.h ../src/2.72.3-eda9959d65.clean/gio/gdbuserror.h ../src/2.72.3-eda9959d65.clean/gio/gdbusaddress.h ../src/2.72.3-eda9959d65.clean/gio/gdbusconnection.h ../src/2.72.3-eda9959d65.clean/gio/gdbusmessage.h ../src/2.72.3-eda9959d65.clean/gio/gdbusnameowning.h ../src/2.72.3-eda9959d65.clean/gio/gdbusnamewatching.h ../src/2.72.3-eda9959d65.clean/gio/gdbusproxy.h ../src/2.72.3-eda9959d65.clean/gio/gdbusintrospection.h ../src/2.72.3-eda9959d65.clean/gio/gdbusmethodinvocation.h ../src/2.72.3-eda9959d65.clean/gio/gdbusserver.h ../src/2.72.3-eda9959d65.clean/gio/gdbusinterface.h ../src/2.72.3-eda9959d65.clean/gio/gdbusinterfaceskeleton.h ../src/2.72.3-eda9959d65.clean/gio/gdbusobject.h ../src/2.72.3-eda9959d65.clean/gio/gdbusobjectskeleton.h ../src/2.72.3-eda9959d65.clean/gio/gdbusobjectproxy.h ../src/2.72.3-eda9959d65.clean/gio/gdbusobjectmanager.h ../src/2.72.3-eda9959d65.clean/gio/gdbusobjectmanagerclient.h ../src/2.72.3-eda9959d65.clean/gio/gdbusobjectmanagerserver.h ../src/2.72.3-eda9959d65.clean/gio/gtestdbus.h gio/gnetworking.h
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "builtin-baseline": "065656f4fc8ce29ada4158147d93a12f0907bee6",
  "dependencies": [
    {
      "name": "eigen3",
      "version>=": "3.3.9"
    },
    {
      "name": "glew",
      "version>=": "2.2.0"
    },
    {
      "name": "cgal",
      "version>=": "5.4"
    },
    {
      "name": "boost",
      "version>=": "1.79.0"
    },
    {
      "name": "glm",
      "version>=": "0.9.9.8"
    },
    {
      "name": "opencv3",
      "version>=": "3.4.16"
    },
    {
      "name": "tbb",
      "version>=": "2020_U3"
    },
    {
      "name": "osg",
      "version>=": "3.6.5"
    },
    {
      "name": "spdlog",
      "version>=": "1.10.0"
    },
    {
      "name": "metis",
      "version>=": "2022-07-27"
    },
    {
      "name": "nanort",
      "version>=": "2019-08-20"
    },
    {
      "name": "gdal",
      "version>=": "3.5.1"
    },
    {
      "name": "liblas",
      "version>=": "1.8.1"
    },
    {
      "name": "cryptopp",
      "version>=": "8.6.0"
    }
  ],
  "name": "v3d-dependency",
  "overrides": [
    {
      "name": "eigen3",
      "port-version": 0,
      "version-string": "3.3.9"
    },
    {
      "name": "glew",
      "port-version": 0,
      "version-string": "2.2.0"
    },
    {
      "name": "cgal",
      "port-version": 0,
      "version-string": "5.4"
    },
    {
      "name": "boost",
      "port-version": 0,
      "version-string": "1.79.0"
    },
    {
      "name": "glm",
      "port-version": 1,
      "version-string": "0.9.9.8"
    },
    {
      "name": "opencv3",
      "port-version": 4,
      "version-string": "3.4.16"
    },
    {
      "name": "tbb",
      "port-version": 8,
      "version-string": "2020_U3"
    },
    {
      "name": "osg",
      "port-version": 13,
      "version-string": "3.6.5"
    },
    {
      "name": "spdlog",
      "port-version": 1,
      "version-string": "1.10.0"
    },
    {
      "name": "metis",
      "port-version": 0,
      "version-string": "2022-07-27"
    },
    {
      "name": "nanort",
      "port-version": 2,
      "version-string": "2019-08-20"
    },
    {
      "name": "gdal",
      "port-version": 4,
      "version-string": "3.5.1"
    },
    {
      "name": "liblas",
      "port-version": 9,
      "version-string": "1.8.1"
    },
    {
      "name": "cryptopp",
      "port-version": 0,
      "version-string": "8.6.0"
    }
  ],
  "version-semver": "1.0.0"
}

```
</details>
[package-x64-linux-dynamic-dbg-out.log](https://github.com/microsoft/vcpkg/files/12288267/package-x64-linux-dynamic-dbg-out.log)

--- a/ports/gettext/portfile.cmake
+++ b/ports/gettext/portfile.cmake
@@ -153,6 +153,13 @@ if(subdirs)
     vcpkg_install_make()
     vcpkg_copy_pdbs()
     vcpkg_copy_tool_dependencies("${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin")
+    file(GLOB unix_runtime LIST_DIRECTORIES false
+        "${CURRENT_PACKAGES_DIR}/lib/libgettext*${VCPKG_TARGET_SHARED_LIBRARY_SUFFIX}*"
+        "${CURRENT_PACKAGES_DIR}/lib/libtextstyle*${VCPKG_TARGET_SHARED_LIBRARY_SUFFIX}*"
+    )
+    if(unix_runtime)
+        file(INSTALL ${unix_runtime} DESTINATION "${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin")
+    endif()
     file(GLOB link_libs LIST_DIRECTORIES false "${CURRENT_PACKAGES_DIR}/lib/*" "${CURRENT_PACKAGES_DIR}/bin/*.dll")
     file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/include" ${link_libs})
 endif()

--- a/ports/gettext/vcpkg.json
+++ b/ports/gettext/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "gettext",
   "version": "0.21.1",
-  "port-version": 2,
+  "port-version": 3,
   "description": "A GNU framework to help produce multi-lingual messages.",
   "homepage": "https://www.gnu.org/software/gettext/",
   "license": "GPL-3.0-only",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2834,7 +2834,7 @@
     },
     "gettext": {
       "baseline": "0.21.1",
-      "port-version": 2
+      "port-version": 3
     },
     "gettext-libintl": {
       "baseline": "0.21.1",

--- a/versions/g-/gettext.json
+++ b/versions/g-/gettext.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "20e8e3535959bba6066d8887bb1c95e622d86f5a",
+      "version": "0.21.1",
+      "port-version": 3
+    },
+    {
       "git-tree": "0915f060ea0fb91c5a2356f2171de0a714e79437",
       "version": "0.21.1",
       "port-version": 2

